sdks/python/apache_beam/coders/standard_coders_test.py

```diff
@@ -81,7 +81,10 @@ def attribute_parser_from_type(type_):
     # TODO: This should be exhaustive
     type_info = type_.WhichOneof("type_info")
     if type_info == "atomic_type":
-      return schemas.ATOMIC_TYPE_TO_PRIMITIVE[type_.atomic_type]
+      if type_.atomic_type == schema_pb2.BYTES:
+        return lambda x: x.encode("utf-8")
```

Comment:
As I understand we can't use the result of ATOMIC_TYPE_TO_PRIMITIVE[schema_pb2.BYTES] because it would result in `x = bytes('abc')` which would raise an error?

Reply:
That's right. This is basically the same as the logic here:
https://github.com/apache/beam/blob/3b0ba733dc3d9b063b3fff9297073f26c06e8a0d/sdks/python/apache_beam/coders/standard_coders_test.py#L118-L121

I have a TODO in the code to unify these paths but haven't gotten around to it yet.